### PR TITLE
Run Baremetal operator in a loop locally

### PR DESCRIPTION
**Pull request by maelk**
Baremetal operator exits when the cluster is unreachable, that
happens during the tests. Running it in a loop to restart it
if this happens

### CODE REVIEW

**Comment by maelk** [comment]
/hold
trying to increase the minikube CPUs to see if it impacts the problem

File: 03_launch_mgmt_cluster.sh [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -xe
 
+CUR_DIR="$(dirname "$(readlink -f "${0}")")/"
+
```
Comment (hardys) [inline 03_launch_mgmt_cluster.sh]: There is already `$SCRIPTDIR` in lib/common.sh which can be used for this?

**Comment by hardys** [comment]
This seems like a reasonable temporary workaround to unblock CI, but it'd be good to figure out why the minikube VM becomes unresponsive - one nit noted but otherwise lgtm as a workaround.

**Comment by maelk** [comment]
/hold cancel
Increasing the cpus to 4 do not help with the issue, we'll go with this workaround

**Pull request merged**